cmd/kubeadm/app/constants/constants.go

```diff
@@ -472,6 +472,7 @@ var (
 		19: "3.4.13-0",
 		20: "3.4.13-0",
 		21: "3.4.13-0",
+		22: "3.4.14-0",
```

Comment:
```suggestion
		22: "3.4.13-0",
```

let's keep this version the same until the new etcd image is available.

Reply:
@neolit123 updated.